commands/update.rb

```diff
@@ -0,0 +1,59 @@
+require 'json'
+require_relative '../lib/color'
+require_relative '../lib/const'
+require_relative '../lib/package'
+require_relative '../lib/package_utils'
+
+class Command
+  def self.update
+    # Update package lists.
+    Dir.chdir(CREW_LIB_PATH) do
+      # Set sparse-checkout folders.
+      system "git sparse-checkout set packages manifest/#{ARCH} lib commands bin crew tests tools"
+      system 'git sparse-checkout reapply'
+
+      system "git fetch #{CREW_REPO} #{CREW_BRANCH}", exception: true
+      system 'git reset --hard FETCH_HEAD', exception: true
+      system 'git-restore-mtime -sq 2>/dev/null', exception: true if PackageUtils.installed?('git_mestrelion_tools')
```

Comment:
Why is this necessary?

Reply:
I guess this was carried over so...

Reply:
I'm not exactly sure what question you're asking.